conda/environment.yml

```diff
@@ -0,0 +1,182 @@
+    - importlib-metadata==3.7.3
+    - ipykernel==5.5.0
+    - ipython==7.21.0
+    - ipython-genutils==0.2.0
+    - ipywidgets==7.6.3
+    - jedi==0.18.0
+    - jinja2==2.11.3
+    - joblib==1.0.1
+    - jsonschema==3.2.0
+    - jupyter==1.0.0
+    - jupyter-client==6.1.12
+    - jupyter-console==6.4.0
+    - jupyter-core==4.7.1
+    - jupyterlab-pygments==0.1.2
+    - jupyterlab-widgets==1.0.0
+    - keras-preprocessing==1.1.2
+    - kiwisolver==1.3.1
+    - kubernetes==12.0.1
+    - llvmlite==0.36.0
+    - markdown==3.3.4
+    - markupsafe==1.1.1
+    - matplotlib==3.3.4
+    - mistune==0.8.4
+    - mlops-tempo==0.1.0.dev6
+    - mlserver==0.3.1.dev5
+    - mlserver-tempo==0.3.1.dev5
+    - murmurhash==1.0.5
+    - nbclient==0.5.3
+    - nbconvert==6.0.7
+    - nbformat==5.1.2
+    - nest-asyncio==1.5.1
+    - networkx==2.5
+    - notebook==6.3.0
+    - numba==0.53.0
+    - numpy==1.19.5
+    - oauthlib==3.1.0
+    - opencv-python==4.5.1.48
+    - opencv-python-headless==4.5.1.48
+    - opt-einsum==3.3.0
+    - orjson==3.5.1
+    - packaging==20.9
+    - pandas==1.2.3
+    - pandocfilters==1.4.3
+    - parso==0.8.1
+    - pathy==0.4.0
+    - pexpect==4.8.0
+    - pickleshare==0.7.5
+    - pillow==8.1.2
+    - preshed==3.0.5
+    - prometheus-client==0.9.0
+    - prompt-toolkit==3.0.18
+    - protobuf==3.15.6
+    - ptyprocess==0.7.0
+    - pyasn1==0.4.8
+    - pyasn1-modules==0.2.8
+    - pycparser==2.20
+    - pydantic==1.7.3
+    - pygments==2.8.1
+    - pyparsing==2.4.7
+    - pyrsistent==0.17.3
+    - python-dateutil==2.8.1
+    - python-rclone==0.0.2
+    - pytz==2021.1
+    - pywavelets==1.1.1
+    - pyyaml==5.4.1
+    - pyzmq==22.0.3
+    - qtconsole==5.0.3
+    - qtpy==1.9.0
+    - regex==2021.3.17
+    - requests==2.25.1
+    - requests-oauthlib==1.3.0
+    - rsa==4.7.2
+    - sacremoses==0.0.43
+    - scikit-image==0.18.1
+    - scikit-learn==0.24.1
+    - scipy==1.6.1
+    - seldon-deploy-sdk==0.2.1
+    - send2trash==1.5.0
+    - shap==0.37.0
+    - six==1.15.0
+    - slicer==0.0.3
+    - smart-open==3.0.0
+    - soupsieve==2.2.1
+    - spacy==3.0.5
+    - spacy-legacy==3.0.1
+    - spacy-lookups-data==1.0.0
+    - srsly==2.4.0
+    - starlette==0.13.6
+    - tensorboard==2.4.1
+    - tensorboard-plugin-wit==1.8.0
+    - tensorflow==2.4.1
+    - tensorflow-estimator==2.4.0
+    - tensorflow-probability==0.12.1
+    - termcolor==1.1.0
+    - terminado==0.9.3
+    - testpath==0.4.4
+    - thinc==8.0.2
+    - threadpoolctl==2.1.0
+    - tifffile==2021.3.17
+    - tokenizers==0.10.1
+    - toolz==0.11.1
+    - tornado==6.1
+    - tqdm==4.59.0
+    - traitlets==5.0.5
+    - transformers==4.4.2
+    - typer==0.3.2
+    - typing-extensions==3.7.4.3
+    - urllib3==1.26.4
+    - uvicorn==0.13.4
+    - wasabi==0.8.2
+    - wcwidth==0.2.5
+    - webencodings==0.5.1
+    - websocket-client==0.58.0
+    - werkzeug==1.0.1
+    - widgetsnbextension==3.5.1
+    - wrapt==1.12.1
+    - zipp==3.4.1
+prefix: /home/clive/anaconda3/envs/tempo-examples
```

Comment:
Have the feeling we should skip this line.

Reply:
yes - will update - good catch